### PR TITLE
PFM-74 Add validation to profile and remove location in building

### PR DESCRIPTION
**Pull request by SamDudley**

### CODE REVIEW

File: peoplefinder/models.py [hunk 1]
```diff
@@ -63,23 +63,70 @@ class Person(models.Model):
     country = models.ForeignKey(
         "Country", models.SET_DEFAULT, default=Country.get_default_id, related_name="+"
     )
-    workdays = models.ManyToManyField("Workday", blank=True, related_name="+")
+    workdays = models.ManyToManyField(
+        "Workday",
+        verbose_name="Which days do you usually work?",
+        blank=True,
+        related_name="+",
```
Comment (rossmiller) [inline peoplefinder/models.py:70]: Nice!

File: peoplefinder/templates/peoplefinder/components/checkboxes.html [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+        {% for checkbox in field %}
+            <div class="govuk-checkboxes__item">
+                {{ checkbox.tag }}
+                <label class="govuk-label govuk-checkboxes__label" for="{{ checkbox.id_for_label }}">
+                    {{ checkbox.choice_label }}
+                </label>
+            </div>
+        {% endfor %}
+    </div>
 </div>
```
Comment (rossmiller) [inline peoplefinder/templates/peoplefinder/components/checkboxes.html:24]: Do we care about EOF lines in templates?

**Review by rossmiller — APPROVED** [review]
LGTM! A couple of comments that can safely be ignored.

File: peoplefinder/templates/peoplefinder/components/checkboxes.html (hunk 2 above)
Comment (SamDudley) [reply peoplefinder/templates/peoplefinder/components/checkboxes.html:24]: I'll leave it in for now for consistency reasons but happy to discuss this and change project wide.

**Pull request merged**